i18n/zh/docusaurus-plugin-content-docs/current/rollout/appdeploy.md

```diff
@@ -181,13 +182,13 @@ You must run all commands in that directory.
    example-app-v2
    ```
 
-1. Then use the two AppRevisions to update the AppDeployment:
+5. 加下来我们用两个应用来更新 AppDeployment 对象::
 
    ```bash
    $ kubectl apply -f appdeployment-2.yaml
    ```
 
-   (Optional) If you have Istio installed, you can apply the AppDeployment with traffic split:
+   （可选）如果你已安装Istio，你可以将 AppDeployment 与 traffic split 一起应用:
```

Comment:
```suggestion
   （可选）如果你已安装了 Istio，你可以将 AppDeployment 结合流量切分的功能一起使用:
```

Reply:
spaces are added and traffic splits is translated.